### DISCUSSION TITLE
Pi-Hole SSH Password Not Working

### DISCUSSION THREAD

**Discussion by AuTune** [tteck/Proxmox#3169] [Help Needed]
Every time I try to set up a Pi-Hole container I put in my password that I have saved in a password manager that will auto type it for me and I know for certain it's typing it right. It doesn't let me log in. Not into the web UI, not into the shell. This happens every time I try to spin one up, across 10 different generated passwords I've tried. They're all 20 character letters, upper and lower, and numbers. No symbols.

**Comment by Mellowlynx** [comment]
Normally, it will use auto login for the shell, just open it through the Proxmox UI or SSH.
After the installation, you need to set a password for the Web UI by opening the LXC shell and running `pihole -a -p`

**Comment by dada051** [comment]
I had the same problem yesterday. SSH login was not working for a recently created Pi-Hole LXC. So I made a backup with teleporter, and spun up a new Pi-Hole LXC. Works for me now. I hope it will works forever.
